crates/autopilot/src/domain/quote/mod.rs

```diff
@@ -1,9 +1,10 @@
 use {super::OrderUid, crate::domain::eth};
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
Not sure that all types that can implement `Copy` should also implement `Copy`.
Usually it's a good idea to be conscious about copying lots of data and this struct is definitely not small.